ui/app/services/wizard.js

```diff
@@ -4,44 +4,9 @@ import Service, { inject as service } from '@ember/service';
 import { Machine } from 'xstate';
 
 import getStorage from 'vault/lib/token-storage';
-
-import TutorialMachineConfig from 'vault/machines/tutorial-machine';
-import SecretsMachineConfig from 'vault/machines/secrets-machine';
-import PoliciesMachineConfig from 'vault/machines/policies-machine';
-import ReplicationMachineConfig from 'vault/machines/replication-machine';
-import ToolsMachineConfig from 'vault/machines/tools-machine';
-import AuthMachineConfig from 'vault/machines/auth-machine';
-
-const TutorialMachine = Machine(TutorialMachineConfig);
+import { STORAGE_KEYS, DEFAULTS, MACHINES } from 'vault/helpers/wizard-constants';
```

Comment:
Not really a necessary change, but you could further destructure off of `STORAGE_KEYS` here for the other `const`s if you wanted: 

`const {TUTORIAL_STATE, FEATURE_LIST, <other consts here> } = STORAGE_KEYS`